src/wp-includes/l10n/class-wp-translation-controller.php

```diff
@@ -0,0 +1,410 @@
+<?php
+/**
+ * I18N: WP_Translation_Controller class.
+ *
+ * @package WordPress
+ * @subpackage I18N
+ * @since 6.5.0
+ */
+
+/**
+ * Class WP_Translation_Controller.
+ *
+ * @since 6.5.0
+ */
+class WP_Translation_Controller {
```

Comment:
Does this need to be extended? If not:

```suggestion
final class WP_Translation_Controller {
```